pkg/cmd/root.go

```diff
@@ -34,9 +35,9 @@ func Root(p cli.Params) *cobra.Command {
 	pflag.CommandLine = pflag.NewFlagSet(os.Args[0], pflag.ExitOnError)
 
 	var cmd = &cobra.Command{
-		Use:   "tkn",
-		Short: "CLI for tekton pipelines",
-		Long:  ``,
+		Use:                    "tkn",
```

Comment:
This changes is weird (aka I don't see why `goimports` or `gofmt` would do that)